docs/scripts/build.sh

```diff
@@ -17,6 +17,14 @@ if [ -n "$NETLIFY" ]; then
   cd ..
   echo Working dir $(pwd)
 
+  # Build bb.js
+  cd barretenberg/ts
```

Comment:
Could this follow the same pattern as build_package?

Reply:
Yes, I can update. I don't think this builds though. 

Are you able to build docs (`yarn build` in `./docs/`) on your machine?

Reply:
I have just run it ok on the mainframe, does netlify have a docker container build option, if its not building there its probably due to the lack of required dependencies like clang16 etc.